src/tbl_opts.h

```diff
@@ -1,24 +1,9 @@
-/*
-  Table of command-line options
-
-  Copyright (c) 2021 Reuben Thomas <rrt@sc3d.org>
-
-  This file is part of libpaper.
-
-  This program is free software: you can redistribute it and/or modify it
-  under the terms of the GNU General Public License as published by
-  the Free Software Foundation; either version 3 of the License, or (at
-  your option) any later version.
-
-  This program is distributed in the hope that it will be useful,
-  but WITHOUT ANY WARRANTY; without even the implied warranty of
-  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-  GNU General Public License for more details.
+// SPDX-FileCopyrightText: 2021 Reuben Thomas <rrt@sc3d.org>
+//
+// SPDX-License-Identifier: GPL-3.0-or-later
 
-  You should have received a copy of the GNU General Public License
-  along with this program.  If not, see
-  <https://www.gnu.org/licenses/gpl-3.0.html>.
-*/
+// Table of command-line options
+// This file is part of libpaper.
```

Comment:
If we're not keeping "This file is part of libpaper" elsewhere, no need to keep it here. But shouldn't there be some header specifying of what program each file is a part?

Reply:
I have no strong opinion on this. I think it does not hurt to keep 5 and 6